apps/prairielearn/src/ee/auth/lti13/lti13Auth.ts

```diff
@@ -308,11 +308,11 @@ async function verify(req: Request, tokenSet: TokenSet) {
 
   // Check nonce to protect against reuse
   const nonceKey = `lti13auth-nonce:${req.params.lti13_instance_id}:${lti13_claims['nonce']}`;
-  const cacheResult = await cacheGet(nonceKey);
+  const cacheResult = await cache.get(nonceKey);
   if (cacheResult) {
     throw error.make(500, 'Cannot reuse LTI 1.3 nonce, try login again');
   }
-  cacheSet(nonceKey, true, 60 * 60 * 1000); // 60 minutes
+  cache.set(nonceKey, true, 60 * 60 * 1000); // 60 minutes
```

Comment:
This returns a promise; let's make sure to await it for proper error handling.

```suggestion
  await cache.set(nonceKey, true, 60 * 60 * 1000); // 60 minutes
```

Reply:
Updated to await for this in [dacc892](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/dacc892c32a53e09450e4b8ba921a9242e3bce7f).